packages/dialog/test/header-footer.common.js

```diff
@@ -502,14 +500,14 @@ describe('header/footer feature', () => {
       });
 
       it('should set overflow attribute when scrollbar appears on resize', async () => {
-        overlay.style.maxHeight = '300px';
+        overlay.style.maxHeight = '270px';
```

Comment:
Changed hardcoded values here due to replacing Lumo version of the overlay with the one from `src` folder.